cli/command/registry/login.go

```diff
@@ -41,7 +41,7 @@ func NewLoginCommand(dockerCli command.Cli) *cobra.Command {
 	flags := cmd.Flags()
 
 	flags.StringVarP(&opts.user, "username", "u", "", "Username")
-	flags.StringVarP(&opts.password, "password", "p", "", "Password")
+	flags.StringVarP(&opts.password, "password", "p", "", "Password (deprecated, use --password-stdin)")
```

Comment:
https://docs.docker.com/engine/#feature-deprecation-policy says we simply need to label it and let it sit for 3+ releases. I don't mind waiting a while, but it seems good to tell people it's bad.

In any case. I've dropped those bits from this patch.